pkg/webhook/instance_admission_test.go

```diff
@@ -40,6 +41,10 @@ func TestValidateUpdate(t *testing.T) {
 					Name:    "bar",
 					Trigger: update,
```

Comment:
Is this actually allowed? I thought an update is only performed if the OV changes, and doesn't have anything to do with param changes?

Reply:
I don't think we established these rules. It's all a bit fuzzy and determined by some old code. 🤷

Reply:
Then we should establish some rules to remove the fuzziness. We shouldn't have legacy code that drives behavior we don't fully understand in a project that's still in beta.